README.md

```diff
@@ -25,122 +25,95 @@ Apollo-android features:
 
 The latest Gradle plugin version is [ ![Download](https://api.bintray.com/packages/apollographql/android/apollo/images/download.svg) ](https://bintray.com/apollographql/android/apollo-gradle-plugin/_latestVersion)
 
-To use this plugin, add the dependency to your project's root build.gradle file:
+In your app Gradle file, apply the `com.apollographql.apollo` plugin and add the Apollo dependencies:
 
 ```groovy
-buildscript {
-  repositories {
-    jcenter()
-  }
-  dependencies {
-    classpath("com.apollographql.apollo:apollo-gradle-plugin:x.y.z")
-  }
+// app/build.gradle or app/build.gradle.kts
+plugins {
+  id("com.apollographql.apollo").version("x.y.z")
 }
-```
-
-Then add the dependencies to your app's build.gradle and apply file and apply the `com.apollographql.apollo` plugin:
-
-```groovy
-apply plugin: 'com.apollographql.apollo'
 
 repositories {
-    jcenter()
+  jcenter()
 }
 
 dependencies {
   implementation("com.apollographql.apollo:apollo-runtime:x.y.z")
   
-  // If not already on your classpath, you might need the jetbrains annotations
-  compileOnly("org.jetbrains:annotations:13.0")
-  testCompileOnly("org.jetbrains:annotations:13.0")
+  // optional: if you want to use the normalized cache
```

Comment:
Similar to these, what about adding the api too? We can have a comment like: 

```
// optional: if you do not want to depend on runtime, add apollo-api for generated code.
```

It would be really good for awareness